ColorCodeGeneration.cpp

```diff
@@ -0,0 +1,25 @@
+#include "ColorCodeGeneration.h"
+
+namespace ColorCode{
+
+    const char* MajorColorsList[] = {"White", "Red", "Black", "Yellow", "Violet"};
+    int MajorColorsCount = sizeof(MajorColorsList) / sizeof(MajorColorsList[0]);
+    const char* MinorColorsList[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+    int MinorColorsCount = sizeof(MinorColorsList) / sizeof(MinorColorsList[0]);
+
+    ColorPair GetColorFromValue(int colorCodeValue) {
+        int indexValue = colorCodeValue - 1;
+        primaryColor majorColor = (primaryColor)(indexValue / MinorColorsCount);
+        secondaryColor minorColor = (secondaryColor)(indexValue % MinorColorsCount);
+        return ColorPair(majorColor, minorColor);
+    }
+    int GetValueFromColor(primaryColor major, secondaryColor minor) {
```

Comment:
getValue... does not indicate what value. The function name can be more meaningful